katharsis-jpa/src/main/java/io/katharsis/jpa/internal/meta/impl/MetaAttributeProjectImpl.java

```diff
@@ -0,0 +1,5 @@
+package io.katharsis.jpa.internal.meta.impl;
+
+public class MetaAttributeProjectImpl {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this empty class, write its code or make it an "interface". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2094)